jest.config.js

```diff
@@ -55,6 +55,12 @@ config.coverageThreshold = {
         branches: 60,
         functions: 0,
         lines: 23
+    },
+    './*/error-handler/**/*.js': {
+        statements: 45,
+        branches: 60,
+        functions: 40,
+        lines: 47
```

Comment:
These thresholds aren't great as it is. We shouldn't lower them any further. Can the test coverage be improved to at least maintain the level?

Reply:
see comment above